pallets/xcmp-queue/src/lib.rs

```diff
@@ -334,6 +465,17 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// Inbound aggregate XCMP messages. It can only be one per ParaId.
```

Comment:
The second sentence is not so clear to me, as we can have multiple messages per ParaId, since we have a bounded vec.

I would repharse it to something like:

```suggestion
	/// Inbound aggregate XCMP messages per ParaId
```